docs/src/modules/concepts/pages/concepts.adoc

```diff
@@ -172,46 +172,54 @@ Akka provides a basket of interoperable components that can be used to design an
 
+Akka supports two primary ways for components to interact, either with each other or with the outside world.
 
 [cols="1,3", options="header"]
 [.compact]
 |===
 |Client Type |Description
 
 |xref:java:component-and-service-calls.adoc[ComponentClient]
-|One component can directly invoke another component. Akka treats these direct invocations as non-blocking, asynchronous messages, meaning that the invoking component immediately returns after making the invocation and can invoke any component regardless of its location.
+|One component can invoke another using a direct call. These invocations are non-blocking and asynchronous. The invoking component continues its work without waiting for a reply. This helps maintain responsiveness in the system.
 
-For example, a Workflow component handling a long-running business transaction can invoke an Agent component to perform a task, and the Workflow can continue processing without waiting for the Agent to complete its work.
+A common example is a Workflow that performs a long-running process and invokes an Agent to handle a specific task. The Workflow can move on while the Agent does its work in parallel.
 
 |Events
-|Components can emit events that can be subscribed to by other components. For example, an entity can emit an event when its state is updated. Other components can subscribe to this event, similar to how clients subscribe to Kafka topics. Events are propagated transparently between components through reliable, brokerless messaging.
+|Components can emit events to signal that something has occurred. Other components may subscribe to these events. This model resembles traditional publish-subscribe systems but does not require external brokers.
 
-Components can also subscribe to events that are incoming from external sources, which can be 3rd party brokers, APIs, or real-time streams of data.
+For example, when an Entity updates its state, it can emit an event. A View can subscribe to that event to stay in sync. Events can also come from external sources, such as APIs or streaming services.
 |===
 
-Akka is designed so that components remain loosely coupled and can communicate with each other regardless of where they run. Inter-component communication runs on virtual threads managed by the Akka runtime. These threads are lightweight and isolated in a way that ensures system resources remain available for other tasks. This allows components to interact without interfering with each other or affecting overall responsiveness. This basic behavior is one reason why Akka services, whether stateful or stateless, can scale to 10M transaction per second (TPS).
+Akka encourages building systems where components are loosely coupled. Communication between them runs on virtual threads, which are lightweight and isolated. This allows each interaction to happen independently, helping to avoid contention and ensuring that system resources are used efficiently.
```

Comment:
This is not completely correct. The application code is running on virtual thread, which can be suspended when communicating with other components. The communication itself (by the runtime) is not running on virtual threads.

Reply:
But maybe that's too much detail to tell for this document, no? 
If we explain that the blocking operations is done in a controlled way, that's enough.